bioinf_tools.py

```diff
@@ -1 +1,130 @@
+from modules import fastq_module
+from modules import protein_module
+from modules import dna_module
+from typing import Union, Dict, List, Tuple
+import sys
 
+
```

Comment:
по PEP8 сначала идут импорты из стандартной библиотеки, потом из внешних библиотек, потом из собственных. Все группы импортов разделяются пустой строкой, между импортами и кодом - две пустях строки
(кстати, сначала полные импорты, потом частичные, так что выглядеть ээто должно так):
```suggestion
import sys
from typing import Union, Dict, List, Tuple

from modules import fastq_module
from modules import protein_module
from modules import dna_module



```